src/_guides/language/language-tour.md

```diff
@@ -651,9 +651,9 @@ usually just call them *lists*.
 Dart list literals look like JavaScript array literals. Here’s a simple
 Dart list:
 
-<!-- language-tour/list-literal/bin/main.dart -->
+<?code-excerpt "language-tour/list-literal/bin/main.dart"?>
 {% prettify dart %}
-var list = [1, 2, 3];
+List<int> list = [1, 2, 3];
```

Comment:
Ditto: IMHO, this type annotation helps emphasize that we are in the section talking about Lists.

Reply:
I would just leave `var`.

Reply:
Ok.